api/configs.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  jwtSecret: process.env.JWT_SECRET || "Whoa there cowboy, why you peeking up my secrets?!"
```

Comment:
If in production and there's no secret, throwing an error might be a good idea, so it can't start in an unsecured mode.